testgrid/config/config.yaml

```diff
@@ -2411,3 +2429,24 @@ dashboards:
     test_group_name: ci-kubernetes-node-kubelet-1.7
   - name: federation-build-1.7
     test_group_name: ci-kubernetes-federation-build-1.7
+
+- name: cos-image-validation
+  dashboard_tab:
+  - name: cos_beta-k8s-new
```

Comment:
use underscore (`k8s_blah`) instead of hyphen (`k8s-blah`), the same as with `cos_blah`. I think Yang is asking the thing up above ;)